includes/polling/os/netspire.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$oids = array('netspireDeviceModelName.0', 'netSpireDeviceDeviceSerialNo.0');
+
+$data = snmp_get_multi_oid($device, $oids, '-OUQs', 'OACOMMON-MIB', 'openaccess');
+
+$hardware = $data[$oids[0]];
```

Comment:
This is easily broken I think. Take a look at how it's done in `includes/polling/os/ipolis.inc.php`